Marlin/Marlin_main.cpp

```diff
@@ -4945,7 +4944,7 @@ inline void gcode_M226() {
   }
 #endif
  
-#ifdef E_REGULATOR
+#if ENABLED(E_REGULATOR)
   /**
    * M236 - Send Value to ADC w/ no EEPROM write
    */
```

Comment:
I believe M236 has a bunch of pneumatic-related things. Maybe it should also have `&& ENABLED(PNEUMATICS)`